content/en/news/releases/1.15.x/_index.md

```diff
@@ -0,0 +1,46 @@
+---
+title: 1.15.x Releases
+description: Announcements for the 1.15 release and its associated patch releases.
+weight: 15
+list_by_publishdate: true
+layout: release-grid
+decoration: dot
+---
+
```

Comment:
The first 8 lines need to stay here, but the rest of the doc goes into content/en/news/releases/1.15.x/announcing-1.15/_index.md (so one level deeper in the tree).

You can copy the first few lines from 1.14 for example. I don't think you need aliases at this point.

Reply:
Fixed file structure and moved one level deeper.